modules/sonarr_manager.py

```diff
@@ -293,6 +296,49 @@ async def list_user_shows(update: Update, context: ContextTypes.DEFAULT_TYPE):
     return CHOOSE_SHOW_TO_MANAGE
 
 
+async def list_tv_schedules(update: Update, context: ContextTypes.DEFAULT_TYPE):
+    logger.debug(f'Firing callback: {inspect.stack()[0][3]}')
+    query = update.callback_query
+    try:
+        context.user_data.is_configured('sonarr_hostname')
+    except ValueError:
+        await context.bot.send_message(
+            chat_id=update.message.from_user.id,
+            text="You have not configured a server.Exiting."
+        )
+        return ConversationHandler.END
+    tag = next(filter(lambda x: x.get("label") == f'tg:{update.effective_user.id}',
```

Comment:
You need a check here for `if tag` or change the default parameter to be literal like `{ seriesIds: [] }` so your loop doesn't fail.